cou/steps/backup.py

```diff
@@ -44,9 +44,12 @@ async def backup(model: Model) -> Path:
     await model.run_on_unit(unit_name, f"chmod o+rx {basedir}")
 
     local_file = COU_DATA / os.path.basename(remote_file)
-    logger.info("SCP from  mysql-innodb-cluster:%s to %s ...", remote_file, local_file)
+    logger.info("SCP from mysql-innodb-cluster:%s to %s ...", remote_file, local_file)
     await model.scp_from_unit(unit_name, remote_file, str(local_file))
 
+    progress_indicator.info(f"Backed up MySQL to {local_file}")
+    progress_indicator.start()
```

Comment:
actually can we not touch the progress indicator here? It doesn't seem right for a function to mess with the progress indicator, when it isn't fully managing the indicator (it prints a message and starts the spinner, but nothing else, so it's incomplete).

I'm happy if this simply uses `print()` (like elsewhere in the codebase), or the message be defined higher in the call chain so it's picked up by the existing spinner.

Reply:
> actually can we not touch the progress indicator here? It doesn't seem right for a function to mess with the progress indicator, when it isn't fully managing the indicator (it prints a message and starts the spinner, but nothing else, so it's incomplete).

I guess that will be a disaster to use `print` when progress indicator is still running. Elsewhere in code base using `print` is when the progress indicator is not running.

And according to description of `spinner.info` [here](https://pypi.org/project/halo/):

```
Stops the spinner and changes symbol to ℹ. If text is provided, it is persisted else current text is persisted. Returns the instance.
```

So the behavior in this PR is the `.info()` will stop the spinner, print one line with symbol ℹ then `spinner.start()` will restart the spinner with previous text. And I think it's a much standard way to use *halo* library compare to using `print`.

Reply:
Ok, that's fair. It's not great separation of responsibilities, but I guess we can't improve the situation until we change how we manage output/logs/spinners. Could you add a comment above it to explain why the seemingly-random calls to progress_indicator instead of other methods there?

Reply:
Comment added